### PR TITLE
Add blog on Chrys Woods's talk.

### PR DESCRIPTION
**Pull request by r-j-arnold**

### CODE REVIEW

File: _posts/2019-01-30-Chris-woods-talk.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+![*Image courtesy of Nic Trott*](/assets/images/chrys-woods-talk-foss.png){: .img-fluid}
+
+**By [Becky Arnold](https://software.ac.uk/fellows/becky-arnold),
+[University of Sheffield](https://www.sheffield.ac.uk).**
+
+On the 30th of January 2019, [Christopher Woods](https://chryswoods.com/), an EPSRC fellow at the [Advanced Computing Research Centre](https://www.bristol.ac.uk/acrc/) at the [University of Bristol](http://www.bristol.ac.uk/homepage/) gave a talk on "How To Design And Engineer Good Code For Research" at the University of Sheffield. Around seventy people attended to hear Chris talk about what makes code good, how to go about writing it, and how to make sure it’s reliable. The slides can be found [here](https://drive.google.com/file/d/1CBTAhCVixccui1DjeUT13qh6ga5SDXjl/view).
+
+Chris first discussed the question of what does good code for research need to be, and gave three main criteria. 
+
+It needs to be flexible enough to adapt to evolving research questions
```
Comment (willfurnass) [inline _posts/2019-01-30-Chris-woods-talk.md]: Make into a bulletted list

File: _posts/2019-01-30-Chris-woods-talk.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+He went on to discuss different kinds of test, such as unit tests to check small pieces of code are working as expected, and systems tests which check that the pieces are working together correctly. Chris particularly advocated for runtime tests which test key aspects of the program as it runs. This allows problems to be caught early on, effectively “Minimising the blast radius”. Without runtime tests such errors will propagate until they either break something vital further on (making them difficult to trace and fix) or until the program ends without the user ever being aware that their result are compromised. Chris ended his talk with a piece of homework for everyone: write at least one test.
```
Comment (willfurnass) [inline _posts/2019-01-30-Chris-woods-talk.md]: "at least one test when they get back to the office"?

**Review by willfurnass — CHANGES_REQUESTED** [review]
Two suggestions but, apart from those, LGTM.  Thanks for the post!

File: _posts/2019-01-30-Chris-woods-talk.md (hunk 1 above)
Comment (r-j-arnold) [reply _posts/2019-01-30-Chris-woods-talk.md]: Done. I

File: _posts/2019-01-30-Chris-woods-talk.md (hunk 2 above)
Comment (r-j-arnold) [reply _posts/2019-01-30-Chris-woods-talk.md]: Done

**Comment by willfurnass** [comment]
@r-j-arnold The site can't build unless you enter a valid date in the YAML front matter of the Markdown doc.

**Comment by r-j-arnold** [comment]
> @r-j-arnold The site can't build unless you enter a valid date in the YAML front matter of the Markdown doc.

@willfurnass Yeah, I just noticed that, hopefully it'll pass now

**Comment by r-j-arnold** [comment]
@willfurnass Passed travis, and requested changes have been made

**Pull request merged**